tslib.es6.js

```diff
@@ -104,7 +104,7 @@ export function __generator(thisArg, body) {
 }
 
 export function __exportStar(m, exports) {
-    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
+    for (var p in m) if (!exports.hasOwnProperty(p)) __createBinding(exports, m, p);

```

Comment:
`exports.hasOwnProperty(p)` won’t work in case `exports` has a `null` prototype or has overridden `hasOwnProperty(…)`: <https://github.com/microsoft/TypeScript/pull/37013> and <https://github.com/microsoft/tslib/pull/92>.

Reply:
We *should* probably get in the habit of saving off intrinsics like this (i.e. `var hasOwn = Object.prototype.hasOwnProperty` and `hasOwn.call(exports, p)`), but we don't necessarily want to have to introduce a `__hasOwn` helper for that kind of case.

That said, its up to @weswigham if he wants to take on that change for tslib right now, since it's out of scope for this change.

Reply:
That said, we do call `Object.hasOwnProperty.call` in `__importStar` below this.